src/components/cards/ConstructBuilding.tsx

```diff
@@ -1,123 +0,0 @@
-import React, { Dispatch, SetStateAction } from "react";
-import {
-  Buildings,
-  BuildingType,
-  ResourcePool,
-  Resources,
-  ResourceType,
-} from "../../types/";
-import {
-  cloneBasicObjectWithJSON,
-  updateResources as updateResourcePool,
-} from "../../utils";
-import { resourceChecker } from "../../utils/resourceChecker";
-import { AddRemoveButton } from "../buttons";
-import {
-  CardCostsInfo,
-  CardHeader,
-  CardShowCount,
-  CardSymbol,
-  CardTemplate,
-} from "../cards";
-import CardImageHeaderDescriptionContainer from "./CardImageHeaderDescriptionContainer";
-
-interface ConstructBuildingProps {
-  buildings: Buildings;
-  setBuildings: Dispatch<SetStateAction<Buildings>>;
```

Comment:
This assumes that you will always be passing some react `setState()` function here, which may not always be the case.